pkgs/development/libraries/qscintilla/default.nix

```diff
@@ -1,33 +1,30 @@
 { stdenv, lib, fetchurl, unzip
-, qt4 ? null, qmake4Hook ? null
-, withQt5 ? false, qtbase ? null, qtmacextras ? null, qmake ? null
+, withQt5 ? true, qtbase, qtmacextras ? null
```

Comment:
You can drop `withQt5 ? true` now as it is not used anywhere.

Also `? null` is probably not needed.